### PR TITLE
Protocolupgrade: BUIP061

### PR DESCRIPTION
**Pull request by HeaTeR**
add BUIP 061: Demonstration of Phase in Full Network Upgrade Activated by Miners

### CODE REVIEW

File: 061.mediawiki [hunk 1]
```diff
@@ -0,0 +1,92 @@
+<pre>
+BUIP061: Demonstration of Phase in Full Network Upgrade Activated by Miners
+Proposer: LIN Zheming
+Submitted on: ?
```
Comment (ftrader) [inline 061.mediawiki]: You can set this to 2017-06-12 , it is fine.

Comment (HeaTeR) [inline 061.mediawiki]: Done.

**Comment by HeaTeR** [comment]
@sickpig Sorry about the multiple commits. I'm still learning how to use git. and I don't know how to do the "squash" thing. My bad, sad.

**Comment by sickpig** [comment]
@HeaTeR no prob, it's not a showstopper I'm going to merge it in any case.

In any case if you want to know of to git squash just have a look at this:

https://www.devroom.io/2011/07/05/git-squash-your-latests-commits-into-one/

**Pull request merged**